test/services/dumpers/agent-nodejs/expected/mariadb/env

```diff
@@ -0,0 +1,12 @@
+FOREST_AUTH_SECRET=forestAuthSecret
```

Comment:
Is there any reason why this file is not called `.env` ?

Reply:
The .gitignore will stop them from being tracked.

Reply:
🔴